### ISSUE TITLE
Definition of qini curve is unequal the implemented function

### ISSUE THREAD

**Issue by jroessler** [wayfair/pylift#37]
Hi,

I have a question. In your documentation you claim that qini curve is defined as: 
nt,1 - nc,1 * Nt / Nc. But your implemented function looks like: nt1o1/Nt - nt0o1/Nc.

It looks like there is an indifference between your documentation and code. 

Thanks for your help!

**Comment by rsyi** [comment]
Yeah agree this is confusing. Thanks for pointing it out. This is the typical definition of Qini (as in the Radcliffe paper), but we should definitely just be writing out the definition that we actually use...

Fixed this on https://github.com/df-foundation/pylift (the docs in this repo will be wrong, but the RTD file references the new repo).

**Comment by irhete** [comment]
Hello! Am I correct that your Nt and Nc are the total counts of treated and control cases in the dataset? In that case your definition is different from Radcliffe's original Qini curve definition, where Nc and Nt are the counts within the segment (i.e. up to phi).
